<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0" stdsetdef="1">
  <author></author>
  <comment></comment>
  <exportmacro></exportmacro>
  <class>KLoanPaymentPage</class>
  <widget class="QWidget" name="KLoanPaymentPage">
    <property name="geometry">
      <rect>
        <x>0</x>
        <y>0</y>
        <width>525</width>
        <height>373</height>
      </rect>
    </property>
    <layout class="QVBoxLayout">
      <item>
        <widget class="QLabel" name="qLabel1">
          <property name="focusPolicy">
            <enum>Qt::NoFocus</enum>
          </property>
          <property name="text">
            <string>Now enter the information required for the periodic loan payments. If your regular payments contains any additional fees, click on the &lt;b&gt;Additional fees&lt;/b&gt; button to enter them.</string>
          </property>
          <property name="wordWrap">
            <bool>true</bool>
          </property>
        </widget>
      </item>
      <item>
        <layout class="QHBoxLayout">
          <item>
            <spacer name="spacer10">
              <property name="sizeHint">
                <size>
                  <width>63</width>
                  <height>21</height>
                </size>
              </property>
              <property name="sizeType">
                <enum>QSizePolicy::Expanding</enum>
              </property>
              <property name="orientation">
                <enum>Qt::Horizontal</enum>
              </property>
            </spacer>
          </item>
          <item>
            <layout class="QGridLayout">
              <item row="0" column="0" rowspan="1" colspan="2">
                <widget class="QLabel" name="textLabel1">
                  <property name="text">
                    <string>Principal + Interest</string>
                  </property>
                  <property name="wordWrap">
                    <bool>false</bool>
                  </property>
                </widget>
              </item>
              <item row="0" column="2">
                <widget class="QLabel" name="m_basePayment">
                  <property name="font">
                    <font>
                      <bold>true</bold>
                    </font>
                  </property>
                  <property name="text">
                    <string>xxx</string>
                  </property>
                  <property name="alignment">
                    <set>Qt::AlignVCenter|Qt::AlignRight</set>
                  </property>
                  <property name="wordWrap">
                    <bool>false</bool>
                  </property>
                </widget>
              </item>
              <item row="1" column="0">
                <widget class="QLabel" name="textLabel3">
                  <property name="text">
                    <string>+</string>
                  </property>
                  <property name="wordWrap">
                    <bool>false</bool>
                  </property>
                </widget>
              </item>
              <item row="1" column="1">
                <widget class="QPushButton" name="m_additionalFeesButton">
                  <property name="text">
                    <string>Additional Fees</string>
                  </property>
                </widget>
              </item>
              <item row="1" column="2">
                <widget class="QLabel" name="m_additionalFees">
                  <property name="font">
                    <font>
                      <bold>true</bold>
                    </font>
                  </property>
                  <property name="text">
                    <string>xxx</string>
                  </property>
                  <property name="alignment">
                    <set>Qt::AlignVCenter|Qt::AlignRight</set>
                  </property>
                  <property name="wordWrap">
                    <bool>false</bool>
                  </property>
                </widget>
              </item>
              <item row="2" column="0" rowspan="1" colspan="2">
                <widget class="QLabel" name="textLabel5">
                  <property name="text">
                    <string>= periodical payments</string>
                  </property>
                  <property name="wordWrap">
                    <bool>false</bool>
                  </property>
                </widget>
              </item>
              <item row="2" column="2">
                <widget class="QLabel" name="m_totalPayment">
                  <property name="font">
                    <font>
                      <bold>true</bold>
                    </font>
                  </property>
                  <property name="text">
                    <string>xxx</string>
                  </property>
                  <property name="alignment">
                    <set>Qt::AlignVCenter|Qt::AlignRight</set>
                  </property>
                  <property name="wordWrap">
                    <bool>false</bool>
                  </property>
                </widget>
              </item>
            </layout>
          </item>
          <item>
            <spacer name="spacer15">
              <property name="sizeHint">
                <size>
                  <width>62</width>
                  <height>21</height>
                </size>
              </property>
              <property name="sizeType">
                <enum>QSizePolicy::Expanding</enum>
              </property>
              <property name="orientation">
                <enum>Qt::Horizontal</enum>
              </property>
            </spacer>
          </item>
        </layout>
      </item>
      <item>
        <spacer name="spacer12">
          <property name="sizeHint">
            <size>
              <width>20</width>
              <height>20</height>
            </size>
          </property>
          <property name="sizeType">
            <enum>QSizePolicy::Expanding</enum>
          </property>
          <property name="orientation">
            <enum>Qt::Vertical</enum>
          </property>
        </spacer>
      </item>
    </layout>
  </widget>
  <layoutdefault spacing="6" margin="11"/>
  </ui>
